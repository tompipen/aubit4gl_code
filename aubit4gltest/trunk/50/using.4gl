main
call test1()
call test2()
end main


function test1()
define a char(10)
define c char(20)
define b integer
let b=1
let a="####&"
let c=b using "####& ",":"
display "c using ####& string   -", c
let c=b using a,":"
display "c using ####& variable -", c
end function




function test2()
define num_x int 
define string_x char(200) 

let num_x=24 

let string_x=num_x using "john && ",num_x using "j &&" 

display string_x clipped
display "john 24 j 24" 
end function

