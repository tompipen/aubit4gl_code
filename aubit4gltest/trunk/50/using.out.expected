c using ####& string   -    1 :             
c using ####& variable -    1     :         
john 24 j 24
john 24 j 24
